### PR TITLE
refactor(alerts): move channels to alerts

### PR DESCRIPTION
**Pull request by H4ad**
## Pull Request

---

This PR moves the Notification Channels to be under Alerts, before Configuration Tab.

I still kept the `Notification Channels` under settings, but we can eventually remove it from there after few releases.

https://github.com/user-attachments/assets/d620ef4f-3898-41fc-a45b-c41e04f1512a

When viewer:

image

Fixes https://github.com/SigNoz/engineering-pod/issues/5019

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [ ] 🐛 Bug fix
- [x] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

- Tests added/updated: Yes
- Manual verification: Yes
- Edge cases covered: -

---

- Blast radius: Notification Channels
- Potential regressions: Bad Redirect / Unknown
- Rollback plan: Find and fix the issue specifically.

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Feature |
| Description | The Notifications Channels now was moved to be under `Alerts` page. |

---

### 📋 Checklist
- [x] Tests added or explicitly not required
- [x] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

### CODE REVIEW

**Review by AshwinBhatkal — DISMISSED** [review]

File: frontend/jest.config.ts [hunk 1]
```diff
@@ -48,13 +48,14 @@ const config: Config.InitialOptions = {
...
...k-extension-gfm-autolink-literal|micromark-extension-gfm-footnote|micromark-extension-gfm-strikethrough|micromark-extension-gfm-table|micromark-extension-gfm-tagfilter|micromark-extension-gfm-task-list-item|micromark-factory-destination|micromark-factory-label|micromark-factory-space|micromark-factory-title|micromark-factory-whitespace|micromark-util-character|micromark-util-chunked|micromark-util-classify-character|micromark-util-combine-extensions|micromark-util-decode-numeric-character-reference|micromark-util-decode-string|micromark-util-encode|micromark-util-html-tag-name|micromark-util-normalize-identifier|micromark-util-resolve-all|micromark-util-sanitize-uri|micromark-util-subtokenize|micromark-util-symbol|micromark-util-types|decode-named-character-reference|remark-rehype|mdast-util-to-hast|unist-util-position|trim-lines|unist-util-visit|unist-util-visit-parents|unist-util-is|unist-util-generated|mdast-util-definitions|property-information|hast-util-whitespace|space-separated-tokens|comma-separated-tokens|rehype-raw|hast-util-raw|hast-util-from-parse5|devlop|hastscript|hast-util-parse-selector|vfile-location|web-namespaces|hast-util-to-parse5|zwitch|html-void-elements)/)',
```
Comment (AshwinBhatkal) [inline frontend/jest.config.ts:55]: is this related to this change?

Comment (H4ad) [reply frontend/jest.config.ts:55]: Caused by `react-markdown`, we never had any test related to `channels` and now I'm importing it on Alerts, their componets is being imported, and this lib is causing issues with our test setup (esm<->commonjs)

**Review by therealpandey — DISMISSED** [review]

**Review by therealpandey — APPROVED** [review]

**Review by AshwinBhatkal — APPROVED** [review]
